sidnov/a32nx
# feat: payload and fuel UI override

Fixes #[issue_no]

## Summary of Changes

Payload UI Changes

- Overrides the Passenger and Cargo in-game UI. Although UI values will persist as a function of the slider, station weights will be exclusively set by the MCDU AOC menu (and in the future, the EFB menu).

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): TazX [Z+2]#0405

## Testing instructions


- Check all Station weights are initialized to 0 (use the simvar watcher).
- Mess around with the Payload and/ or Fuel sliders within the in-game UI. Check values stay at 0.
- Go to MCDU->ATSU->AOC->W/B (page 2)
- Enter PAX (i.e. 10 pax) and Cargo (i.e. 1.0 Tm), and Start Boarding
- While the boarding process takes place, mess around again with the Payload/ Fuel sliders. Check that the boarding process continues.
- Once the Boarding is completed, check final Aircraft weight is as supposed and calculated
- Mess around again with sliders; check Station weights are maintained


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
